xbmc/interfaces/legacy/ModuleXbmc.cpp

```diff
@@ -529,6 +529,11 @@ namespace XBMCAddon
       return CSysInfo::GetUserAgent();
     }
 
+    std::vector<int> getMousePosition()
```

Comment:
this can't be exposed to the outside. the call is routed to gfx context and windowManager that grab gfx lock